src/main/java/frc/robot/subsystems/AlgaeDestage.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import static edu.wpi.first.wpilibj2.command.Commands.run;
+
+import com.chopshop166.chopshoplib.logging.LoggedSubsystem;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.maps.subsystems.patterns.AlgaeDestageMap;
+import frc.robot.maps.subsystems.patterns.AlgaeDestageMap.Data;
+
+public class AlgaeDestage extends LoggedSubsystem<Data, AlgaeDestageMap> {
+
+    public SmartMotorController motor;
+
+    public static double destageSpeed = 1.0;
```

Comment:
Constants (with static) should be like the following: 
```suggestion
    public static double DESTAGE_SPEED = 1.0;
```